resources/views/company/Company.blade.php

```diff
@@ -42,18 +38,18 @@
                         </tr>
                     </thead>
                     <tbody>
-                        @foreach ($data as $company)
+                        @foreach ($companies as $company)
                             <tr
                                 class="bg-white border-b dark:bg-gray-800 dark:border-gray-700 hover:bg-gray-50 dark:hover:bg-gray-600 h-28">
                                 <th class="w-4 p-4">
                                     @if (count($company->users) > 0)
                                         <div class="flex">
                                             <img class="h-10 w-10"
```

Comment:
sao lại hiển thị thông tin của user 0?

Reply:
- Do yêu cầu bài toán anh ạ : 
+Trong trường hợp công ty không có user_account thì để trống company_account. Ngược lại sẽ hiển thị user đầu tiên
![image](https://github.com/zinza-academy/php-intern--duynd12/assets/85564998/ec91a7fa-1efb-4e99-bb66-e851f4af20c5)